site/blog/2024/01/happy-2024.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 2023 in review and 2024 plans and wishes
+date: 2024-01-17
+excerpt: |
+   Last year passed by quickly, and now it's time to reflect on the highlights of 2023 and outline plans for 2024.
+sidebar: false
+---
+
+<Post>
+
+The past year's efforts were mainly focused on the Access Management tools. Both [Wren:DS](https://wrensecurity.org/projects/wrends/) and [Wren:AM](https://wrensecurity.org/projects/wrenam/) have undergone significant upgrades and security hardening to meet the standards earlier set by [Wren:IDM](https://wrensecurity.org/projects/wrenidm/). This means that the key components are now independent of the legacy infrastructure and have also been polished to ensure the high level of security and robustness. This progress paved the way for the introduction of new features, such as Cisco DUO authentication module.
+
+On top of that there was regular maintenance and improvements in the development infrastructure, e.g. cleanup of the UI libraries.
+To highlight our dedication to delivering dependable software, we have become a CVE numbering authority in the [CVE.org](http://cve.org/) Program, allowing us to transparently manage and share potential security vulnerabilities (see Security Policy). The importance of maintaining an active online presence led us to create an X (Twitter) account providing a more convenient way to share updates. [Follow us](https://twitter.com/WrenSecurity) to keep up with the latest news.
```

Comment:
Security Policy -> Disclosure Policy. And please add hyperlink to https://wrensecurity.org/community/disclosure.html